draft-ietf-jsonpath-base.md

```diff
@@ -1111,8 +1110,12 @@ filter selector iterates over the array or object.
 ##### Existence Tests
 {: unnumbered}
 
-A singular path by itself in a Boolean context is an existence test which yields true if the path selects a node and yields false if the path does not select a node.
-This existence test — as an exception to the general rule — also works with nodes with structured values.
+A path by itself in a Boolean context is an existence test which yields true if the path selects at least one node and yields false if the path does not select a node.
+
+Existence tests are different to comparisons in that:
```

Comment:
Interestingly, North Americans say "different from" not "different to". Possibly the RFC editor will impose North American style?

Reply:
We should aim for North American English. I changed the wording to "Existence tests differ from comparisons in that:" which I think reads well in either British or North American English.